io.github.mlm.asteroids_revenge.yml

```diff
@@ -0,0 +1,32 @@
+app-id: io.github.mlm.asteroids_revenge
+runtime: org.freedesktop.Platform
+runtime-version: '23.08'
+sdk: org.freedesktop.Sdk
+command: asteroids_revenge.x86_64
+finish-args:
+  - --share=ipc
+  - --socket=x11
+  - --socket=pulseaudio
+  - --device=dri
+
+modules:
+  - name: AsteroidsRevenge
+    buildsystem: simple
+    build-commands:
+      - install -D asteroids_revenge.x86_64 /app/bin/asteroids_revenge.x86_64
```

Comment:
how does that work on aarch64?

Reply:
oh sry, i had added the flathub.json but didnt push it before running my test and reverting